CHANGELOG.rst

```diff
@@ -2,6 +2,20 @@
 Changelog
 =========
 
+1.0.1 (2022-08-16)
+------------------
+
+* Fixed a major bug where Python ``int`` and ``float`` types were handled with
+  32-bit instead of 64-bit precision. This caused certain `int` values to be
+  rejected and other `float` values to be rounded incorrectly.
+  Closes `#13 <https://github.com/questdb/py-questdb-client/issues/13>`_.
```

Comment:
```suggestion
* Fixed a major bug where Python ``int`` and ``float`` types were handled with
  32-bit instead of 64-bit precision. This caused certain ``int`` values to be
  rejected and other ``float`` values to be rounded incorrectly.
  Closes `#13 <https://github.com/questdb/py-questdb-client/issues/13>`_.
```